godtools/App/Features/Dashboard/Presentation/Home/FavoritesContentView.swift

```diff
@@ -51,9 +51,8 @@ struct FavoritesContentView: View {
                         
                         FavoriteToolsView(viewModel: viewModel.favoriteToolsViewModel, width: width, leadingPadding: leadingTrailingPadding)
                             .listRowInsets(EdgeInsets())
-                        
-                        Spacer()
-                        
+                            .padding(.bottom, 23)
```

Comment:
Hey @rachaelblue this fix is good for now so we can get a release out. 
I would really like to see if we can come up with a solution where the bottom padding is consistent between tabs without the need to add a specific bottom padding depending on the tab.  Can you create a ticket for that work?  I don't think it's high priority either.

Reply:
Having a default content inset on the dashboard view would be preferred.  That way it would apply to any tabbed view added to the dashboard.